grpc/src/test/java/com/linecorp/armeria/server/grpc/GrpcServiceServerTest.java

```diff
@@ -849,6 +855,7 @@ public void grpcWeb_error() throws Exception {
                 GrpcTestUtil.uncompressedFrame(GrpcTestUtil.requestByteBuf())).aggregate().get();
         assertThat(response.headers()).contains(entry(GrpcHeaderNames.GRPC_STATUS, "10"),
                                                 entry(GrpcHeaderNames.GRPC_MESSAGE, "aborted call"));
+        requestLogQueue.take();
```

Comment:
I wonder if there isn't a more automatic way of handling this. It's a bit tedious to deal with request log queue in tests that aren't testing the request logs. Can we add the `isEmpty` assertion to the tests that actually care about it and in `TearDown` clear the queue instead?

Reply:
It is not possible unfortunately due to timing issues. `RequestLog` may be put into the queue after the test execution is finished. Because tear-down method has no prior information about how many `RequestLog` a test method will produce, it has to wait like up to N seconds to ensure that, which will increase the test run time quite a bit.